docs/_docs/pages.md

```diff
@@ -20,7 +20,16 @@ and associated URLs might look like:
 └── contact.html  # => http://example.com/contact.html
 ```
 
-If you have a lot of pages, you can organize them into subfolders. The same subfolders that are used to group your pages in our project's source will exist in the `_site` folder when your site builds.
+If you have a lot of pages, you can organize them into subfolders. The same subfolders that are used to group your pages in our project's source will exist in the `_site` folder when your site builds. Each individual page will then be generated in a sub folder mathcing its file name and be called index.html
+
+```sh
+.
+|-- about.md          # => http://example.com/about.html
+|-- Documentation     # folder containing pages
+    └── Doc1.md       # => http://example.com/Documentation/Doc1/index.html
```

Comment:
If `about.md` generates `/about.html`, then `Documentation/Doc1.md` will generate `/Documentation/Doc1.html` unless there is a `permalink` entry in the page's front matter.
Please update the comment and paragraph above.

Reply:
Fixed as requested. spending too long playing with Jekyll now which has the other structure, sorry .